<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddEmployee</class>
 <widget class="QDialog" name="DialogAddEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Addition of Employee</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>367</width>
     <height>174</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_Username">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_Type">
      <item>
       <property name="text">
        <string>Banker A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Banker B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Housing Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Car Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Life Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer Scientist</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Lastname"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Firstname">
      <property name="text">
       <string>* First Name : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Type">
      <property name="text">
       <string>* Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Lastname">
      <property name="text">
       <string>* Last Name :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_OnlyIT">
      <property name="text">
       <string>Only for Computer Scientist :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Password">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Firstname"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Username">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_Password">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_required">
      <property name="text">
       <string>* Required</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushBtn_reset">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Firstname</tabstop>
  <tabstop>lineEdit_Lastname</tabstop>
  <tabstop>comboBox_Type</tabstop>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
